runbooks/runbook.md

```diff
@@ -0,0 +1,124 @@
+# UPP - PAC Annotations Mapper
+
+Processes content annotations into UPP from TagMe via PAC.
+
+## Primary URL
+
+<https://upp-prod-delivery-glb.upp.ft.com/__pac-annotations-mapper/>
+
+## Service Tier
+
+Platinum
+
+## Lifecycle Stage
+
+Production
+
+## Delivered By
+
+content
+
+## Supported By
+
+content
+
+## Known About By
+
+- elitsa.pavlova
+- kalin.arsov
+- miroslav.gatsanoga
+- ivan.nikolov
+- marina.chompalova
+- mihail.mihaylov
+- boyko.boykov
+- donislav.belev
+- dimitar.terziev
+
+
+## Host Platform
+
+AWS
+
+## Architecture
+
+Mapper for transforming PAC annotations to UPP annotations:
+
+- Reads PAC metadata for an article from the Kafka topic *NativeCmsMetadataPublicationEvents*
+- Metadata for sources different than `http://cmdb.ft.com/systems/pac` is skipped by this mapper
+- Filters and transforms it to UPP standard JSON representation
+- Writes the result onto the Kafka topic *ConceptAnnotations*
+
+This service has NO service endpoints.
+
+## Contains Personal Data
+
+No
+
+## Contains Sensitive Data
+
+No
+
+## Dependencies
+
+- upp-kafka
+
+## Failover Architecture Type
+
+ActiveActive
+
+## Failover Process Type
+
+FullyAutomated
+
+## Failback Process Type
+
+FullyAutomated
+
+## Failover Details
+
+The service is deployed in both Delivery clusters. The failover guide for the cluster is located here:
+<https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/delivery-cluster>
+
+## Data Recovery Process Type
+
+NotApplicable
+
+## Data Recovery Details
+
+The service does not store data, so it does not require any data recovery steps.
+
+## Release Process Type
+
+PartiallyAutomated
+
+## Rollback Process Type
+
+Manual
+
+## Release Details
+
+Manual failover is needed when a new version of the service is deployed to production. Otherwise, an automated failover is going to take place when releasing. For more details about the failover process please see: <https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/delivery-cluster>
```

Comment:
We don't need failover and automatic failover is not going to take place on release.

Reply:
That's why I needed a review from your team. :) I cloned this repo for the first time now for adding the runbook. :)